services/integration/src/routes/index.js

```diff
@@ -23,4 +24,13 @@ Router.use(
   ObjectivesRouter
 );
 
```

Comment:
You are breaking the patter here.  Remove this space line

Reply:
are you kidding?